web-client/src/ducks/requests/functions.ts

```diff
@@ -1,31 +1,18 @@
-import firebase from 'firebase';
+// import firebase from 'firebase';
```

Comment:
Removed unused observe function and replaced get(Pending/Ongoing/Open/Archivng)Post functions with getFindPosts, getMyCavPosts, and getMyPinPosts.   getMyPinRequestPosts is working.  getMyCavPosts still callls the Cloud function

Possible approach for getMyCavPosts:
- query offers where cavUserRef is current user
- get a list of unique requestIds.
- pass that lis to firebase.
- at some point: if more than 10, do 10 at a time.